src/common/location_summaries.ts

```diff
@@ -0,0 +1,29 @@
+import { Level } from 'common/level';
+import { Metric } from 'common/metric';
+import LocationSummariesJSON from 'assets/data/summaries.json';
+import { findStateFipsCode } from 'common/locations';
+
+export interface MetricSummary {
+  value: number | null;
+  level: Level;
+}
+
+export interface LocationSummary {
+  level: Level;
+  metrics: {
+    [metric in Metric]?: MetricSummary;
```

Comment:
ohhhh fascinating typescript, does this make it so that not all Metrics are required?

Reply:
Yeah!  It took a lot of Googling to find, but it made my day.